.github/workflows/pull-request-target.yml

```
@@ -0,0 +1,58 @@
+          comment-tag: pr_review_policy
+          message: |
+            ## Our Pull Request Approval Process
+
+            This PR will be reviewed according to the [Palisadoes Contributing Guidelines](https://developer.palisadoes.org/docs/contributor-guide/contributing) 
+
+            Thanks for contributing!
+
+      - name: Greeting Message to User
+        uses: actions/first-interaction@v3
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
```

actionlint (1.7.9)

input "repo-token" is not defined in action "actions/first-interaction@v3". available inputs are "issue_message", "pr_message", "repo_token"

(action)